### PR TITLE
STUDIO-2240 Add speaker, battery, charger icons

### PR DESCRIPTION
**Pull request by badiuoanaalexandra**
https://mapbox.atlassian.net/browse/STUDIO-2240

image
image
image

### CODE REVIEW

File: src/svgs/battery.svg
Comment (miafan23) [inline src/svgs/battery.svg:1]: Is this exported from the figma file? It is a bit strange that the icon is not in the middle of the canvas 🤔

File: src/svgs/charger.svg
Comment (miafan23) [inline src/svgs/charger.svg:1]: Why are charger.svg and speaker.svg are white by default?

File: src/svgs/battery.svg
Comment (badiuoanaalexandra) [reply src/svgs/battery.svg:1]: yeah it is, what do you mean it is not in the middle? 
image

Comment (badiuoanaalexandra) [reply src/svgs/battery.svg:1]: It has viewBox="0 0 8 14" , and other ones have 18 18, but at least when showing the icon it looks like it is in the middle ? If I change that then it doesn't show in the middle anymore

Comment (badiuoanaalexandra) [reply src/svgs/battery.svg:1]: Maybe I should ask Stephen to export them with viewbox of 18 18?

File: src/svgs/charger.svg
Comment (badiuoanaalexandra) [reply src/svgs/charger.svg:1]: It just exported it like that, I removed the fill

File: src/svgs/battery.svg
Comment (badiuoanaalexandra) [reply src/svgs/battery.svg:1]: Where do you see that it is not in the middle? At least if I run the app it they all look correct?

Comment (miafan23) [reply src/svgs/battery.svg:1]: image
I mean the left side looks having 2 pixel padding

Comment (badiuoanaalexandra) [reply src/svgs/battery.svg:1]: aaaah.. didn't even notice this.. I'll try to see if I can export it without that

Comment (badiuoanaalexandra) [reply src/svgs/battery.svg:1]: can you check again? I exported it with viewBox="0 0 16 16" (a bit above than before)

**Review by miafan23 — APPROVED** [review]

**Pull request merged**